specifications/api/swagger.yaml

```
@@ -941,6 +941,19 @@ x-assetMint: &assetMint
         Positive values mean creation and negative values mean
         destruction.
 
+x-assetMintedBurnt: &assetMintedBurnt
```

Suggestion: use "burned", rather than "burnt".

Elsewhere, we have used the "burned" terminology; it's probably good to be consistent.

```suggestion
x-assetMintedBurned: &assetMintedBurned
```